tests/bpftool_tests/bpftool_tests.cpp

```diff
@@ -124,6 +124,8 @@ TEST_CASE("prog load map_in_map", "[prog][load]")
     REQUIRE(result == 0);
 }
 
+#if 0
+// TODO(#2974): Once the libbpf APIs are implemented in the xdp-for-windows repo, these tests should be migrated.
```

Comment:
I think the libbpf apis themselves stay in this repo, though they may just fail unless xdp-for-windows is installed.  There is a project to create separate library for xdp user mode apis, and if we port that to windows, that can be in the xdp-for-windows repo.

Reply:
I have updated the comment to mention just XDP functionality, not necessarily migrating the libbpf APIs.